CLI/src/CLI.cs

```diff
@@ -133,14 +133,14 @@ private ReturnCode Compile()
                 analyzerProvider.AddProvider(customAnalyzerProvider);
             }
 
-            //Normalize TypeCobolOptions, the parser does not need to go beyond SemanticCheck for the first phase
+            //Normalize TypeCobolOptions, the parser does not need to go beyond AST for the first phase
```

Comment:
'AST building' would be clearer in this sentence

Reply:
Done